assets/terraform-modules/bootkube/assets.tf

```diff
@@ -166,3 +166,12 @@ resource "local_file" "kubeconfig-admin-named" {
 locals {
   api_servers_external = split(",", join(",", var.api_servers_external) == "" ? join(",", var.api_servers) : join(",", var.api_servers_external))
 }
+
+# Populate chart values file named packet-ccm.yaml.
+resource "local_file" "packet-ccm" {
```

Comment:
This will be installed in the AWS platform as well right? We should control as per platform no?

Reply:
yes, will do it